test/utilities.jl

```diff
@@ -37,3 +57,47 @@ end
 
     @test_throws ArgumentError consecutive_subspans(aligned, Millisecond(1))
 end
+
+@testset "consecutive_overlapping_subspans" begin
```

Comment:
this whole testset is untouched from the private package in which it was copied from

Reply:
(new stuff added in https://github.com/beacon-biosignals/AlignedSpans.jl/pull/19/commits/8eabc1e3787ef765264f3ab83b6213e8806ffb22)